src/components/FormHeaderWrapper.tsx

```diff
@@ -1,15 +1,24 @@
 import type { ReactNode } from 'react';
 
-interface Properties {
+interface FormHeaderWrapperProperties {
+  crumbTrailMarginTop?: boolean;
```

Comment:
It's not clear from the prop name what exactly this flag is for.  
Is this to include additional margin when there *is* a crumb trail?